### PR TITLE
Support and compatibility with official openSUSE repository

### PR DESCRIPTION
**Pull request by cabelo**
@ilya-lavrenov 
Thank you very much for your patience and support.

It took a while to complete the official publication in the science repository of the openSUSE Tumbleweed distribution.

The result is gratifying, as it was a lot of work to make Intel and openSUSE technical policies compatible (it took 3 months of work).

Evidence: 
Upcoming tasks
- Add 15.4. 15.5 and 15.6 (This will be faster)
- Add to default repository in progress.

https://build.opensuse.org/request/show/1168442
https://software.opensuse.org/package/openvino
https://software.opensuse.org/download/package?package=openvino&project=science

### CODE REVIEW

File: docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+Run the following command:
+
+.. code-block:: sh
+
+   zypper se -i openvino
+
+.. note::
+
+   You can additionally install Python API using one of the alternative methods (:doc:`conda <openvino_docs_install_guides_installing_openvino_conda>` or :doc:`pip <openvino_docs_install_guides_installing_openvino_pip>`).
```
Comment (ilya-lavrenov) [inline docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst:105]: as I can see `python3-openvino` is available as RPM package via zypper
so, we need to recommend alternative ways? Suppose no

File: docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+For C++ and C, just run the ``build_samples.sh`` script:
+
+.. tab-set::
+
+   .. tab-item:: C++
+      :sync: cpp
+
+      .. code-block:: sh
+
+         /usr/share/openvino/samples/cpp/build_samples.sh
```
Comment (ilya-lavrenov) [inline docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst:123]: I have installed `openvino-sample` and this command does not work after it

Comment (cabelo) [reply docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst:123]: Done: Intel and openSUSE convention fix

Evidence:  https://build.opensuse.org/request/show/1170090

evidence

File: docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst (hunk 1 above)
Comment (cabelo) [reply docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst:105]: > as I can see `python3-openvino` is available as RPM package via zypper so, we need to recommend alternative ways? Suppose no
> 
> Secondly, I've tried to install `python3-openvino` and I see: image

Let's not do this now. I'm working on the conventions and doing a lot of testing on many hardware platforms.

We are not going to do anything now in this Pull request. How are you @ilya-lavrenov?

As soon as we finish the approval of the Factory phase, I will insert new suggestions here, okay?

File: docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst (hunk 2 above)
Comment (cabelo) [reply docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst:123]: > I have installed `openvino-sample` and this command does not work after it


Done: Intel and openSUSE convention fix

Evidence: https://build.opensuse.org/request/show/1170090

**Comment by ilya-lavrenov** [comment]
@sgolebiewski-intel 
Could you please help to fix documentation build errors?

**Comment by cabelo** [comment]
> @sgolebiewski-intel Could you please help to fix documentation build errors?

@sgolebiewski-intel and @ilya-lavrenov  Everything has been tested and the fix will be published soon, and the patch has already been approved internally.

Evidence: https://build.opensuse.org/request/show/1170178

**Comment by ilya-lavrenov** [comment]
> > @sgolebiewski-intel Could you please help to fix documentation build errors?
> 
> @sgolebiewski-intel and @ilya-lavrenov Everything has been tested and the fix will be published soon, and the patch has already been approved internally.
> 
> Evidence: https://build.opensuse.org/request/show/1170178

I mean the job to build docs has failed:
image

**Comment by cabelo** [comment]
> > > @sgolebiewski-intel Could you please help to fix documentation build errors?
> > 
> > 
> > @sgolebiewski-intel and @ilya-lavrenov Everything has been tested and the fix will be published soon, and the patch has already been approved internally.
> > Evidence: https://build.opensuse.org/request/show/1170178
> 
> I mean the job to build docs has failed: image

Sorry...

**Comment by cabelo** [comment]
> > > @sgolebiewski-intel Could you please help to fix documentation build errors?
> > 
> > 
> > @sgolebiewski-intel and @ilya-lavrenov Everything has been tested and the fix will be published soon, and the patch has already been approved internally.
> > Evidence: https://build.opensuse.org/request/show/1170178
> 
> I mean the job to build docs has failed: image

If I didn't get it wrong, I think I fixed it.

File: docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+      zypper lr |grep -i science
+
+   You will see the available list of packages.
+
+   To list available OpenVINO packages, use the following command:
+
+   .. code-block:: sh
+
+   zypper se openvino
```
Comment (sgolebiewski-intel) [inline docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst:74]:
```suggestion
   .. code-block:: sh

      zypper se openvino
```

File: docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+
+2. Verify that the new repository is set up properly.
+
+   .. code-block:: sh
+
+      zypper lr |grep -i science
+
+   You will see the available list of packages.
+
+   To list available OpenVINO packages, use the following command:
```
Comment (sgolebiewski-intel) [inline docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst:70]:
```suggestion
To list available OpenVINO packages, use the following command:
```

File: docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+
+   .. code-block:: sh
+
+   zypper se openvino
+
+Step 2: Install OpenVINO Runtime Using the YUM Package Manager
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
```
Comment (sgolebiewski-intel) [inline docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst:77]:
```suggestion
Step 2: Install OpenVINO Runtime Using the ZYPPER Package Manager
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
```

File: docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst [hunk 6]
```diff
@@ -0,0 +1,193 @@
...
+
+Install OpenVINO Runtime
+-------------------------
+
+.. tab-set::
+
+   .. tab-item:: The Latest Version
+      :sync: latest-version
+
```
Comment (sgolebiewski-intel) [inline docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst:86]:
```suggestion
```

File: docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst [hunk 7]
```diff
@@ -0,0 +1,193 @@
...
+      Run the following command:
+
+      .. code-block:: sh
+
+         sudo zypper install openvino
```
Comment (sgolebiewski-intel) [inline docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst]:
```suggestion
Run the following command:

.. code-block:: sh

   sudo zypper install openvino
```

**Comment by sgolebiewski-intel** [comment]
> If I didn't get it wrong, I think I fixed it.

I have applied few minor fixes regarding formatting and reference.

**Comment by cabelo** [comment]
> > If I didn't get it wrong, I think I fixed it.
> 
> I have applied few minor fixes regarding formatting and reference.

Thanks

File: docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+
+      zypper lr |grep -i science
+
+   You will see the available list of packages.
+
+To list available OpenVINO packages, use the following command:
+
+   .. code-block:: sh
+
+      zypper se openvino
```
Comment (sgolebiewski-intel) [inline docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst:74]:
```suggestion
.. code-block:: sh

   zypper se openvino
```

File: docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst [hunk 9]
```diff
@@ -0,0 +1,187 @@
...
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Install OpenVINO Runtime
+-------------------------
+
+      Run the following command:
+
+      .. code-block:: sh
+
+         sudo zypper install openvino-devel openvino-sample
```
Comment (sgolebiewski-intel) [inline docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst:86]:
```suggestion
Run the following command:

.. code-block:: sh

   sudo zypper install openvino-devel openvino-sample
```

**Review by sgolebiewski-intel — APPROVED** [review]

**Comment by cabelo** [comment]
@sgolebiewski-intel and @ilya-lavrenov 

Hey, I finished the work with version 2024.1.0, everything is being published together.

Evidence.

https://build.opensuse.org/projects/home:cabelo:branches:science/packages/openvino-2024.1.0/repositories/openSUSE_Tumbleweed/binaries

**Comment by ilya-lavrenov** [comment]
build_jenkins

**Pull request merged**